### ISSUE TITLE
JWST profile error

### ISSUE THREAD

**Issue by alexanderbock** [OpenSpace/OpenSpace#1648]
```
6 errors shown on startup:
(E) Asset Failed to initialize asset C:\Users\Astrolab\Downloads\OpenSpace-0.17.-
1\OpenSpace-0.17.-1\data\assets\scene\solarsystem\missions\jwst\jwst.asset
(E) Asset Lua: When initializing C:\Users\Astrolab\Downloads\OpenSpace-0.17.-
1\OpenSpace-0.17.-1\data\assets\scene\solarsystem\missions\jwst\jwst.asset: ...-1\OpenSpace-
0.17.-1\data\assets\util\asset_helper.asset:101: Error loading scene graph node: (ModelLoader)
Error:'No nodes were loaded' while loading model from
file:'C:\Users\Astrolab\Downloads\OpenSpace-0.17.-1\OpenSpace-0.17.-
1\sync\http\jwst_model\1\JWSTFBX.osmodel'
(E) Scene Cannot add scene graph node 'JWSTTrail'. A node with that name already
exists
(E) Scene Could not load scene graph node
(E) Asset Failed to initialize asset C:\Users\Astrolab\Downloads\OpenSpace-0.17.-
1\OpenSpace-0.17.-1\data\assets\scene\solarsystem\missions\jwst\jwst.asset
(E) Asset Lua: When initializing C:\Users\Astrolab\Downloads\OpenSpace-0.17.-
1\OpenSpace-0.17.-1\data\assets\scene\solarsystem\missions\jwst\jwst.asset: ...-1\OpenSpace-
0.17.-1\data\assets\util\asset_helper.asset:101: Error loading scene graph node
```

 -- As reported by Alex E --

**Comment by alexanderbock** [comment]
- No JWST model loaded
- JWST Trail is offset from JWST position by just several tens of meters, JWST Sun Trail
offset from JWST position by several kilometers
- Hubble Ultra Deep field doesn’t show up

**Comment by Roxeena** [comment]
I can't reproduce this, is there any more info on what setup was used to get the errors?

**Comment by alexetgen** [comment]
<img width="691" alt="OS0 17 0_errors" src="https://user-images.githubusercontent.com/84144028/122228339-bbb36300-ce85-11eb-8192-f203a42298a5.PNG">

Here is an image of the error log so there is some context.

The machine used to get the errors is an Alienware Aurora R12, running Windows 10. I can provide more info if you could elaborate a bit on what you mean by "what setup was used".

**Comment by Roxeena** [comment]
Thanks for the additional information. Have you tried to run it several times or just once? If so, did the errors occur every time you tried to run the profile?

**Comment by alexetgen** [comment]
Of course, and I have run it several times (around 5) and it has happened every time.

**Comment by Roxeena** [comment]
Hmm it’s very strange that I don’t get those errors either on master or on 0.17.-1. From what I can see in the log there appears to be a problem with the JWST model. Could you try to delete the cache folder and the synced model to see if that works? The model should be in your sync folder under `http/jwst_model/1/JWSTFBX.osmodel`. This will trigger the model to be downloaded again when you run OpenSpace next time.

**Comment by alexetgen** [comment]
There was no cache folder for JWST, so I deleted the synced model and opened the profile again, but there was just an additional error added. So, I deleted and redownloaded OS version 0.17.-1 and ran the jwst profile and the model is showing up now with no errors. I'm not sure what caused the errors the first time since I used the same link to download OpenSpace both times, but it is in fact working now.
